samples/chartColors.js

```diff
@@ -0,0 +1,9 @@
+window.chartColors = {
```

Comment:
We could generalize this file and move in it all samples helpers (e.g. `randomScalingFactor`)? It would be renamed to `helpers.js` or `utils.js`? Similar to the [utils.js file](/chartjs/chartjs-plugin-deferred/tree/master/samples) in the deferred plugin.